src/core/def/phare_config.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef PHARE_CORE_DEF_PHARE_CONFIG_HPP
+#define PHARE_CORE_DEF_PHARE_CONFIG_HPP
+
+#include <string>
+
+
+#if __has_include("core/def/_gen_sys.hpp")
```

Comment:
few comments to help me re-understand all this more quickly next time.
// if __gen_sys.hpp  exists, it has been generated by the phare_configurator (config.py) 
// if it does not, the  build_config() version below reflects that phare_configurator is off